client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -387,6 +412,21 @@
                   return localize('volumeAndPage');
               }
           }
+
+          revealHiddenNerdyRowContent() {
+              const detailsElement = this.shadowRoot.querySelector('.parallel-item-biblio-info');
+              const nerdyRow = this.shadowRoot.querySelector('.parallel-item-details');
+              const popup = this.shadowRoot.querySelector('#hidden-nerdy-row');
+              if (nerdyRow.clientWidth < nerdyRow.scrollWidth) {
+                  popup.classList.toggle("show");
+                  popup.style.width = nerdyRow.clientWidth-24;
```

Comment:
24 is a magic number - it should be assigned to a properly named variable because in the future this will be hard to understand :).